Tasks/Rabushenka/Tasks2/HomWr-2.py

```diff
@@ -0,0 +1,51 @@
+14:("полтретего"),15:("полчетвертого"),16:("полпятого"),17:("полшестого"),18:("полседьмого"),19:("полвосьмого"),20:("полдевятого"),21:("полдесятого"),
+22:("полодинадцатого"),23:("полдвенаднатого")}
+sl_min = { 1:("одна минута"), 2 :("две менуты"), 3:("три менуты"),4:("четыре менцты"),5:("пять минут"),6:("шесть минут"),7:("семь минут"),
+8:("восемь минут"),9:("девять минут"),10:("десять минут"),11:("одинадцать минут"),12:("двенадцать минут"),13:("тринадцать минут"), 14:("четырнадцать минут"),
+15:("пятнадцоть минут"),16:("шестнадцоть минут"),17:("семнадцоть минут"),18:("восемнадцоть минут"),19:("девятнадцоть минут"),
+20:("двадцать минут"),21:("двадцать одна минута"),22:("двадцать две минуты"), 23:("двадцать три минуты"),24:("двадцать четыре минуты"),25:("двадцать пять минут"),
+26:("двадцать шесть минут"),27:("двадцать сем минут"),28:("двадцать восем минут"),29:("двадцать девять минут"),31:("тридцать одна минута"), 32 :("тридцать две минуты"),
+33:("тридцать три минуты"),34:("тридцать четыре минуты"),35:("тридцать пять минут"), 36:("тридцать шесть минут"),37:("тридцать семь минут"),38:("тридцать восемь минут"),
+39:("тридцать девять минут"), 40:("сорок минут"), 41:("сорок одна минута"),42:("сорок две минут"),43:("сорок три минуты"),44:("сорок четыре минуты")}
+sl_min_45 = { 45:("без пятнадцоти минут"),46 :("без четырнадцати минут"),47:("без тринадцати минут"),48:("без двенадцати минут"),49:("без одинадцати минут"),50:("без десяти минут"),51:("без девяти минут"),
+52:("без восьми минут"),53:("без семи минут"),54:("без шести минут"),55:("без пяти минут"),56:("без четыреx минут"),57:("без трех минут"),58:("без двух минут"),59:("без одной минуты"),}
+sl_hour_45 = { 0:("первого"),1 :("второго"),2:("третего"),3:("четвертого"),4:("пятого"),5:("шестого"),6:("седьмого"),
+7:("восьмого"),8:("девятого"),9:("десятого"),10:("одинадцатого"),11:("двенадцатого"),12:("час"),13:("два"),14:("три"),15:("четыре"),
+16:("пять"),17:("шесть"),18:("семь"),19:("восемь"),20:("девять"),21:("десять"),22:("одинадцать"),23:("двенадцать")}
+sl_hour1_45 = { 0:("первого"),1 :("второго"),2:("третего"),3:("четвертого"),4:("пятого"),5:("шестого"),6:("седьмого"),
+7:("восьмого"),8:("девятого"),9:("десятого"),10:("одинадцатого"),11:("двенадцатого"),12:("первого"),13:("второго"),14:("третего"),15:("четвертого"),
+16:("пятого"),17:("шестого"),18:("седьмого"),19:("восьмого"),20:("девятого"),21:("десятого"),22:("одинадцатого"),23:("двенадцатого")}
+sl = {1: (sl_hour), 2: (sl_min, sl_hour_45), 3: (sl_min_30), 4: (sl_min_45, sl_hour_45), 5: (sl_min, sl_hour1_45)}
+qu_t = input("выбор операции (1-вруную, 2-авт.):")
+if qu_t == "1 вруную"or (qu_t == "1"):
+    qu_t_1 = int(input("введите часы: "))
+    qu_t_2 = int(input("введите минуты: "))
+    print(f'{(qu_t_1)}:{(qu_t_2)}')
+    min = int(qu_t_2)
+    hour = int(qu_t_1)
+elif qu_t == "2 авт." or (qu_t == "2") or (qu_t == "авт"):
```

Comment:
kinda overcomplicated interface logic

Reply:
why is it difficult? It seems to be simple